### ISSUE TITLE
Wire up the coverage options to instanbul

### ISSUE THREAD

**Issue by mjstahl** [bitovi/testee#33]
The problem was attempting to ignore files and directories like "bower_components".  Currently after configuring testee to ignore these files, they are still included in the coverage process.

Here is an example of a local testee config:
```javascript
options: {
    urls: ['http://localhost:8989/test/test.html'],
    browsers: ["phantom"],
    coverage: {
        "ignore": [
	    "bower_components/*"
	]
    }
}
```

@daffl Pointed me to the TODO listed in the below file.  Nothing is being doing with the config object.
https://github.com/bitovi/testee.js/blob/master/lib/server/extensions/coverage.js#L35

**Comment by phillipskevin** [comment]
The coverage is ignoring anything listed in `config.ignore`. Should it be looking at `config.options.coverage.ignore` if it exists?

**Comment by daffl** [comment]
Probably the better spot for it, yes.

**Comment by phillipskevin** [comment]
The config is being read correctly. Some browsers were not reading the content-type correctly, which is fixed in [Fix issue #55 and #56 - Coverage options](https://github.com/bitovi/testee/pull/57). Even after this fix, files that should be ignored are still being instrumented. Debug statements in the coverage converter function don't seem to be called, so it's unclear exactly what's happening.

**Comment by daffl** [comment]
Running it with `DEBUG testee:coverage` doesn't show the debug messages?

**Comment by phillipskevin** [comment]
No. It will show any debug messages in the `when` function, but not in the `converter`.

**Comment by daffl** [comment]
That is odd. To my knowledge it works fine in https://github.com/bitovi/testee/blob/master/lib/host/inject.js#L15. I can have a look though.

**Comment by phillipskevin** [comment]
It does work for inject. I'm not sure why it's not working for coverage.

**Comment by phillipskevin** [comment]
This seems to work in phantomjs, but does not consistently in Chrome and Firefox. Sometimes the debug messages will be shown, but not always.

**Comment by daffl** [comment]
Allright. All done and closed via #57. Props to @phillipskevin for getting this in!
